virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -11,6 +11,20 @@ toc::[]
 
 Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[our CTO Chris Wright's message].
 
+[id="virt-4-14-doc-feedback"]
+== Providing feedback on {VirtProductName} documentation
+
+We appreciate your feedback on our documentation. Let us know how we can improve it.
+
+To provide feedback, perform the following steps:
+
+. Log in to your Red Hat Jira account.
```

Comment:
Question about the procedure. I see that in the bug, there is a link to a [prepopulated form](https://issues.redhat.com/secure/CreateIssueDetails!init.jspa?pid=12323181&issuetype=1&components=12333768&priority=10200&summary=%5BDoc%5D&customfield_12316142). Is there any reason why we can't use this link? I think OCP docs does something similar.